docs/ANALYTICS.md

```diff
@@ -0,0 +1,24 @@
+# SDS Analytics
+
+These samples showcase some commonly performed data visualization of SDS data.
+
+## [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md)
+
+These samples highlight basic operations of Data Views for OCS, including creation, updating, getting data from and deletion of Data Views. Please see the [Dataviews](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/DATA_VIEWS_README.md) readme for more information on this sample.
```

Comment:
This may be personal preference, but I tend to like 'ReadMe' or 'README' as opposed to 'readme'